src/components/ContactListItem/ContactListItem.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import styled from 'styled-components'
+
+export default function ContactListItem({
+  firstName,
+  lastName,
+  company,
+  index,
+}) {
+  return (
+    <li key={index}>
+      <ContactContainer>
+        <SVGWrapper>
+          <svg
```

Comment:
You may want to consider making your svg more accessible- in this instance, perhaps add a 'role' if the icon is important. This section might be helpful! https://css-tricks.com/accessible-svgs/#example-1-standalone-icon-meaningful

Reply:
Thank you for this interesting resource! I will include this in a later step. :)